content/getting-started/guides/index.mdx

```diff
@@ -46,6 +46,27 @@ See a 3-minute tutorial on creating a simple app with Firebase.
   allowfullscreen
 ></iframe>
 
+## Building apps webinar
+
+See our webinar recording on building LiveChat apps.
+
+Agenda:
```

Comment:
Are you sure you don't want to indicate what time in the video you start talking about something?